src/vs/workbench/browser/layout.ts

```diff
@@ -596,6 +596,10 @@ export abstract class Layout extends Disposable implements IWorkbenchLayoutServi
 						case 'remote':
 							viewletId = 'workbench.view.remote';
 							break;
+						// {{SQL CARBON EDIT}} add notebook view container to views
+						case 'notebooks':
```

Comment:
What is this for?

Reply:
And why is this not necessary for the connections viewlet?

Reply:
connections viewlet has their own view extension defined and not using the vscode's view extension. I now removed the notebookExplorer view extension point and am using the views extension point of vscode and added a notebook-view-container which needs to be added here.